tests/test_flow/fakes.py

```diff
@@ -188,15 +192,7 @@ def create_aip_job(body, parent):
         "state": "SUCCEEDED"
     }
 
-    cached_gcs_fs = gcs._cached_gcs_fs
```

Comment:
Can you also get rid of `gcs._cached_gcs_fs` now?

Reply:
That would remove the caching of `fsspec.filesystem("gcs")`? Not exactly sure if we should do this.